### ISSUE TITLE
Issue with special characters in username logging in

### ISSUE THREAD

**Issue by newandreas** [MediaBrowser/emby-theater-windows#399]
Me and my friends recently encountered an issue signing onto my server on a computer named 'RonjaStasjonær'.

The symptoms were that after initial setup, typing in the correct server address resulted in a flash of the loading circle then it dissappears. If you then restart Emby Theater it won't start past the logo splash screen

I've reproduced this on my own computer by changing the name in Windows 10 and got the same result they did.

**Comment by LukePulverenti** [comment]
Hi, so do you mean special characters in the server name and not the user name?

**Comment by newandreas** [comment]
> Hi, so do you mean special characters in the server name and not the user name?

Whoops, I mean as in the name of the computer running Emby Theater, made a mistake in the title of this issue! So that would be device name for the Emby Server.

**Comment by LukePulverenti** [comment]
And when you enter the address into the app, are you using the computer name or the ip address?

**Comment by newandreas** [comment]
> And when you enter the address into the app, are you using the computer name or the ip address?

The user types in my Emby server address in Emby Theater, https://emby.myname.com with port 443 which is behind a nginx reverse proxy.

**Comment by LukePulverenti** [comment]
Ok, so i assume the address they are entering does not have a special character, only the computer name?

**Comment by newandreas** [comment]
That's correct. The setup I have right now works for 4 other people that also are remotely connecting with Emby Theater. I tried to reproduce the problem on my own computer with Emby Theater and it also seemed to stop working completely.

**Comment by LukePulverenti** [comment]
And I assume you're able to use the web app?

**Comment by newandreas** [comment]
Yes.
